W-16139934 AEH sso restructuring and azure content jh (#49)
mulesoft/docs-api-experience-hub: modules/ROOT/pages/user-roles-permissions-and-profiles.adoc

|===
|Permission |Description |Actions
|Exchange Viewers|Assign this permission to the Organization Administrator or the API Experience Hub Administrator who performs the Salesforce linking process. Users with this permission can't add new assets, edit asset portal content, or share an asset with another user.
a|
* View APIs in Exchange
* Import APIs from Exchange to the portal |
|===

== User Roles for Configuring SSO

Configuring an identity provider for API Experience Hub includes multiple personas that perform different user actions. 

> Depending on your organization’s setup, a user can be assigned one or more of the following roles:

Reviewer: ```suggestion
Depending on your organization’s setup, a user can be assigned one or more of these roles:
```

> API Experience Hub Administrator:: 

Reviewer: Consider converting to table to be consistent with Access Management. 
https://confluence.internal.salesforce.com/x/TRRzNQ

> The AEH Admin is responsible for enabling and disabling an identity provider in the API EXperience Hub UI.

Reviewer: Is AEH an acceptable acronym?

> The AEH Admin is responsible for enabling and disabling an identity provider in the API EXperience Hub UI.

Reviewer: ```suggestion
The AEH Admin is responsible for enabling and disabling an identity provider in the API Experience Hub UI.
```

Salesforce System Administrator::

The Salesforce System Administrator is responsible for setting up the identity provider in Salesforce.

Anypoint Organization Administrator::

The Anypoint Organization Administrator is responsible for mapping the identity provider groups with the API Experience Hub profiles using the group mapping functionality of Access Management teams.

Security team::

> Your organization’s security team is responsible for setting up the identity provider with the required user-groups and applications. Your organization’s security team and the Salesforce System Administrator, who manage the identity provider system in an organization, configure the proper login option in Salesforce. The security team and the Org Admin in the main organization might want to configure the same identity provider option in Anypoint Platform, if required. 

Reviewer: ```suggestion
Your organization’s security team is responsible for setting up the identity provider with the required user groups and applications. Your organization’s security team and the Salesforce System Administrator, who manage the identity provider system in an organization, configure the proper login option in Salesforce. The security team and the Org Admin in the main organization can configure the same identity provider option in Anypoint Platform, if required. 
```

+

> The security team and the Org Admin in the main organization configures the mapping between Salesforce and Anypoint Platform. Access Management provides the functionality of adding group mappings for external identity providers using teams. When the group mappings are properly configured, users who log in to the portal are added automatically to the out-of-the-box profile that they belongs to. The default identity provider is provided to create identities in Anypoint Platform for Salesforce users.

Reviewer: ```suggestion
The security team and the Org Admin in the main organization configure the mapping between Salesforce and Anypoint Platform. Access Management provides the functionality of adding group mappings for external identity providers using teams. After the group mappings are properly configured, users who log in to the portal are added automatically to the out-of-the-box profile that they belong to. The default identity provider is provided to create identities in Anypoint Platform for Salesforce users.
```

The Anypoint Organization Administrator is responsible for mapping the identity provider groups with the API Experience Hub profiles using the group mapping functionality of Access Management teams.

Security team::

Your organization’s security team is responsible for setting up the identity provider with the required user-groups and applications. Your organization’s security team and the Salesforce System Administrator, who manage the identity provider system in an organization, configure the proper login option in Salesforce. The security team and the Org Admin in the main organization might want to configure the same identity provider option in Anypoint Platform, if required. 
+
The security team and the Org Admin in the main organization configures the mapping between Salesforce and Anypoint Platform. Access Management provides the functionality of adding group mappings for external identity providers using teams. When the group mappings are properly configured, users who log in to the portal are added automatically to the out-of-the-box profile that they belongs to. The default identity provider is provided to create identities in Anypoint Platform for Salesforce users.

> 

Reviewer 2: thought: since we already released PBV, now the user might also be added to the Custom User Groups.
Maybe we also need to adjust the Profiles table below

Author: I add that info to the table.
